README.rst

```diff
@@ -135,6 +157,12 @@ Theme
 -----
 Tahoe themes are copied and available in ``edx-codebase-theme`` directory near the ``edx-platform``. Refer to `edx-theme-codebase#How to use on Devstack <https://github.com/appsembler/edx-theme-codebase#how-to-use-on-devstack>`_ for more info on setting up your custom theme.
 
+Troubleshooting Theme Installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Refer to instructions in the `How to use on Devstack <https://github.com/appsembler/edx-theme-codebase#how-to-use-on-devstack>`_ section of the ``edx-theme-codebase`` Github repository.
```

Comment:
We seem to have a duplicate information here. Please consider combining the two sections.